### PR TITLE
K8SPG-169 Use percona native user secrets only

### PR DESCRIPTION
**Pull request by cap1984**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| demand-backup | passed |
| upgrade | passed |
| init-deploy | passed |
| smart-update | failed |
| scaling | passed |
| recreate | failed |
| version-service | passed |
| affinity | passed |
| scheduled-backup | passed |
| monitoring | passed |
| self-healing | passed |
| operator-self-healing | passed |
| clone-cluster | failed |
| tls-check | passed |
| users | failed |
| ns-mode | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/198/commits/d5f5357d200de7f2950d857267bd3c366520836e
image: `perconalab/percona-postgresql-operator:PR-198-d5f5357d-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-198-d5f5357d-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-198-d5f5357d-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-198-d5f5357d-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-198-d5f5357d-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-198-d5f5357d-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| demand-backup | passed |
| init-deploy | passed |
| upgrade | passed |
| smart-update | passed |
| scaling | passed |
| version-service | passed |
| recreate | passed |
| affinity | passed |
| scheduled-backup | passed |
| monitoring | passed |
| self-healing | passed |
| operator-self-healing | passed |
| clone-cluster | passed |
| tls-check | passed |
| users | passed |
| ns-mode | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/198/commits/d043c351b9030e49f48c8ec5bf5d624df8e00798
image: `perconalab/percona-postgresql-operator:PR-198-d043c351-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-198-d043c351-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-198-d043c351-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-198-d043c351-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-198-d043c351-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-198-d043c351-pgo-deployer`

File: deploy/users-secret.yaml [hunk 1]
```diff
@@ -8,3 +8,12 @@ stringData:
...
   pguser: pguser_password
+---
+apiVersion: v1
+data:
+  password: YWRtaW4=
+  username: YWRtaW4=
+kind: Secret
+metadata:
+  name: some-name-pmm-secret
+type: Opaque
```
Comment (hors) [inline deploy/users-secret.yaml:19]:
```suggestion
apiVersion: v1
kind: Secret
metadata:
  name: cluster1-pmm-secret
type: Opaque
stringData:
  password: admin
  username: admin
```

Comment (hors) [reply deploy/users-secret.yaml:19]: we need to fix pmm secret in CR

**Review by hors — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| demand-backup | passed |
| upgrade | passed |
| init-deploy | passed |
| smart-update | passed |
| scaling | passed |
| version-service | passed |
| recreate | passed |
| affinity | passed |
| scheduled-backup | failed |
| monitoring | passed |
| self-healing | passed |
| operator-self-healing | passed |
| clone-cluster | passed |
| tls-check | passed |
| users | passed |
| ns-mode | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/198/commits/44b117a2770ba7b1a5116618c0ebe70f338f6d50
image: `perconalab/percona-postgresql-operator:PR-198-44b117a2-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-198-44b117a2-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-198-44b117a2-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-198-44b117a2-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-198-44b117a2-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-198-44b117a2-pgo-deployer`

**Review by hors — APPROVED** [review]

**Pull request merged**